src/mangata/common.js

```diff
@@ -131,8 +142,31 @@ export const scheduleTask = async ({
         const answerPool = await confirm({ message: '\nDo you want to continue to schedule auto-compound. Press ENTRE to continue.', default: true });
 
         if (answerPool) {
+            console.log("\n4. Check if It's need to send foreign token to Turing ...");
+            if (scheduleFeeAsset === mangataDefaultAsset || executionFeeAsset === mangataDefaultAsset) {
```

Comment:
You are comparing two objects, which is not safe in javascript. It’s better to company their keys, which are strings.